backend/spec/requests/answers_request_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+RSpec.describe 'AnswersController', type: :request do
+  describe '#create' do
+    context 'when data is valid' do
+      let(:survey) { create(:survey) }
+      let!(:answers_survey) { create(:answers_survey) }
+      let!(:question) { create(:question) }
+      let!(:option_a) { create(:option) }
+      let!(:option_b) { create(:option) }
+
+      before do
+        answer_params = {
+          question_id: question.id,
+          option_ids: [option_a.id, option_b.id],
+          answers_survey_id: answers_survey.id
+        }
+
+        post api_v1_answers_path, params: answer_params, headers: auth_headers
+      end
+
+      it { expect(response).to have_http_status :created }
+
+      it { expect(Answer.count).to eq(1) }
+
+      it { expect(Answer.first.options.count).to eq(2) }
+
+      it 'matches answer attributes' do
+        expected_attributes = {
+          'id' => anything,
+          'question_id' => question.id,
+          'options' => [{
+            'id' => option_a.id,
+            'name' => option_a.name,
+            'correct' => option_a.correct
+          },
+                        {
```

Comment:
this indentation is a bit strange, does rubocop want it?

Reply:
Actually, rubocop did this xD I was also a bit confused but he is the boss!